util/make_new_dif.py

```diff
@@ -115,8 +115,9 @@ def _load_irqs(self):
         assert (self.hjson_data and
                 "ERROR: must load IP HJSON before loarding IRQs")
         irqs = []
-        for irq in self.hjson_data["interrupt_list"]:
-            irqs.append(Irq(irq))
+        if "interrupt_list" in self.hjson_data:
+            for irq in self.hjson_data["interrupt_list"]:
+                irqs.append(Irq(irq))
```

Comment:
This may need some more work for multi-bit IRQs: see hw/ip/gpio/data/gpio.hjson.
It lists one irq with name `gpio` but sets its width to 32. In my auto-generated IRQ test, I expect the DIF to provide this enum: 
```c
  typedef enum dif_gpio_irq {
    kDifGpioIrqGpio0,
    ...,
    kDifGpioIrqGpio31
  } dif_gpio_irq_t;
```

Reply:
This will be special case, tracking in #8142, and will address in the (future) PR that integrates the autogenerated IRQs into the GPIO block.